core/arch/arm/plat-ti/conf.mk

```diff
@@ -15,6 +15,7 @@ $(call force,CFG_PL310_LOCKED,y)
 $(call force,CFG_SECURE_TIME_SOURCE_REE,y)
 arm32-platform-cpuarch		:= cortex-a9
 else
+CFG_OTP_SUPPORT ?= y
```

Comment:
Hum. All flavors work the same so yeah, I think `CFG_OTP_SUPPORT` is useless.
Time to stop reviewing code and leave the office now ;)